plugins/UM3NetworkPrinting/src/ConfigurationChangeModel.py

```diff
@@ -3,11 +3,16 @@
 
 from PyQt5.QtCore import pyqtSignal, pyqtProperty, QObject, pyqtSlot
 
+BLOCKING_CHANGE_TYPES = [
+    "material_insert", "buildplate_change"
+]
+
 class ConfigurationChangeModel(QObject):
     def __init__(self, type_of_change: str, index: int, target_name: str, origin_name: str) -> None:
         super().__init__()
         self._type_of_change = type_of_change
                                     # enum = ["material", "print_core_change"]
+        self._can_override = False if self._type_of_change in BLOCKING_CHANGE_TYPES else True
```

Comment:
Do you mean `self._can_override = self._type_of_change not in BLOCKING_CHANGE_TYPES` ?

Reply:
Yes.